lib/common.py

```diff
@@ -492,9 +492,13 @@ def is_suspended(self):
     def is_paused(self):
         return self.power_state() == 'paused'
 
-    def start(self):
+    # By xe design on must be an host name-label
+    def start(self, on=None):
         print("Start VM")
-        return self.host.xe('vm-start', {'uuid': self.uuid})
+        args = { 'uuid': self.uuid }
```

Comment:
I think the python codestyle we are more or less following doesn't add spaces between brackets and contents.

Reply:
Is there a python codestyle that we are more or less following? XD

Reply:
Found it: https://www.python.org/dev/peps/pep-0008/#id26

There are several places where this comment applies.